lua51/PSPBUILD

```diff
@@ -1,35 +1,38 @@
 pkgname=lua51
 pkgver=5.1.5
-pkgrel=1
-pkgdesc="Lua programming language for PSP"
+pkgrel=2
+pkgdesc="a powerful, efficient, lightweight, embeddable scripting language (version 5.1)"
 arch=('mips')
 url="https://lua.org"
 license=('MIT')
 depends=()
 makedepends=()
 optdepends=()
-source=(
-"https://www.lua.org/ftp/lua-5.1.5.tar.gz"
-"psp.patch")
-sha256sums=(
-'2640fc56a795f29d28ef15e13c34a47e223960b0240e8cb0a82d9b0738695333'
-'SKIP')
+source=("https://www.lua.org/ftp/lua-${pkgver}.tar.gz")
+sha256sums=('2640fc56a795f29d28ef15e13c34a47e223960b0240e8cb0a82d9b0738695333')
 
 prepare() {
-    tar -xzf lua-5.1.5.tar.gz
-    cd lua-5.1.5
-    patch -p1 < ../psp.patch
+    cd "lua-${pkgver}"
+    sed -i 's/liblua.a/liblua51.a/' Makefile src/Makefile
+
+    # extract the license from readme
+    grep -A21 Copyright doc/readme.html | \
+        sed -e 's/&ndash;/–/g' \
+            -e 's/&copy;/©/g' \
+            -e '/<P>/d' > LICENSE
 }
 
 build() {
-    cd lua-5.1.5
-    make -f Makefile.psp
+    cd "lua-${pkgver}/src"
+    make a CC="psp-gcc" AR="psp-ar rcu" RANLIB="psp-ranlib"
```

Comment:
What does the `a` do?

Reply:
It's a target that only builds `liblua.a` (on in case of patched makefile, `liblua${LUAV}.a`), all other platforms use `all` target instead which includes `lua` and `luac` executables which are not needed on PSP.

Reply:
Specifically, it's defined here: [makefile#L65](https://github.com/lua/lua/blob/4324904b60db5243ede68d0922c1bf3c0dd05986/makefile#L65)

Reply:
Makes sense, I was worried it might be a typo